docs/components/movement-sensor/wheeledodometry.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Configure a wheeledodometry movement sensor"
+linkTitle: "wheeled-odometry"
+type: "docs"
+description: "Configure a wheeled-odometry movement sensor."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "movement sensor"]
+# SMEs: Rand, Martha
+---
+
+Configure a `wheeled-odometry` movement sensor to implement _wheeled odometry_ on your robot.
+
+_Wheeled odometry_ is the estimation of position, orientation, linear velocity, and angular velocity using the dimensions of a base.
+This model uses [encoders](/components/encoder/) from [position reporting motors](/components/motor/gpio/) to get an odometry estimate from a wheeled base.
+
+## Set-up requirements
+
+To prepare your robot, attach [encoders](/components/encoder/) to each of the position-reporting motors on your base to measure their rotation.
```

Comment:
This instruction is specific to the gpio motor model only - I would say "pick out a motor that can report it's position through encoders"

Reply:
Put this on line 25-- lmk if you want me to move to be in that first sentence! I also moved a lot of the info 
i had on the bottom of the page before so that people scanning can see the info about why it's useful more easily-- lmk what you think

Reply:
happy to add a few more sentences like `this is why this is useful` if you want!